conda_index/index/convert_cache.py

```diff
@@ -322,22 +326,22 @@ def merge_index_cache(channel_root, output_db="merged.db"):
             try:
                 with combined_db:
                     combined_db.execute(query, (channel_prefix,))
-            except sqlite3.OperationalError as e:
+            except sqlite3.OperationalError:
                 log.error("OperationalError on %s", query)
                 raise
 
         combined_db.execute("DETACH DATABASE subdir")
 
 
-def test_from_archive(archive_path):
+def _test_from_archive(archive_path):
     conn = common.connect("linux-64-cache.db")
     create(conn)
-    with closing(conn):
+    with closing(conn):  # type: ignore
         convert_cache(conn, extract_cache(archive_path))
         remove_prefix(conn)
 
 
-def test():
+def _test():
```

Comment:
Is this to prevent pytest from finding the function?

Reply:
I don't remember if that was a factor, but I tend to use leading underscores for function that I don't want referenced from an external location.